inhibit/inhibit.go

```diff
@@ -43,6 +42,9 @@ type Inhibitor struct {
 
 // NewInhibitor returns a new Inhibitor.
 func NewInhibitor(ap provider.Alerts, rs []*config.InhibitRule, mk types.Marker, logger log.Logger) *Inhibitor {
+	if logger == nil {
```

Comment:
@simonpasquier As far as I can tell, `logger` is only `nil` for tests, right? I would prefer to create a proper `logger` in the test code and pass it to `NewInhibitor` instead of bloating the real code here. What do you think?

Reply:
> logger is only nil for tests

yes. Agree with your recommendation.